src/components/TopLevelModals/index.tsx

```diff
@@ -28,7 +28,8 @@ export default function TopLevelModals() {
       <ConnectedAccountBlocked account={account} isOpen={accountBlocked} />
       <Bag />
       <UniwalletModal />
-      <UniswapWalletBanner />
+      <BaseAnnouncementBanner />
+      {/* <UniswapWalletBanner /> */}
```

Comment:
Can we delete the old `components/Banner/UniswapWalletBanner` file?

Reply:
No, the new one is just a re-skin for Base specifically. It's only rendered on that network